src/telegram_bot/bot.py

```diff
@@ -1,203 +1,141 @@
+
+    async def start(self, update: Update, _: ContextTypes.DEFAULT_TYPE) -> BOT_STATES:
+        """
+        Starts the conversation
+        """
+        menu_keyboard = InlineKeyboardMarkup(KEYBOARD)
+
+        if update.message:
+            await update.message.reply_text(
+                "Hi! Welcome to the bot. Choose the command to continue",
+                reply_markup=menu_keyboard,
+            )
+        else:
+            self.logger.error("Exception in start")
+
+        return BOT_STATES.ACTION_SELECTION
+
+    async def photo_to_sticker_prompt(self, update: Update, _: ContextTypes.DEFAULT_TYPE) -> BOT_STATES:
+        """Handles the button click and asks for a photo."""
+        if update.callback_query:
+            query = update.callback_query
+            await query.answer()
+            await query.message.reply_text("Please send a photo to prepare sticker.")  # type: ignore[union-attr]
+        else:
+            self.logger.error("Exception in photo_to_sticker_prompt")
+        return BOT_STATES.PHOTO_STICKER
+
+    async def edit_photo_prompt(self, update: Update, _: ContextTypes.DEFAULT_TYPE) -> BOT_STATES:
+        """Handles the button click and asks for a photo."""
+        if update.callback_query:
+            query = update.callback_query
+            await query.answer()
+            await query.message.reply_text("Please send a photo to edit.")  # type: ignore[union-attr]
+        else:
+            self.logger.error("Exception in edit_photo_prompt")
+        return BOT_STATES.PHOTO_EDIT
+
+    async def audio_to_text_prompt(self, update: Update, _: ContextTypes.DEFAULT_TYPE) -> BOT_STATES:
+        """Handles the button click and asks for a photo."""
+        if update.callback_query:
+            query = update.callback_query
+            await query.answer()
+            await query.message.reply_text("Please send audio to transcribe.")  # type: ignore[union-attr]
+        else:
+            self.logger.error("Exception in audio_to_text_prompt")
+        return BOT_STATES.AUDIO
+
+    async def photo_to_sticker(self, update: Update, context: ContextTypes.DEFAULT_TYPE) -> BOT_STATES:
+        """
+        Handler to gathed needed data and call sticker pack creation model
+        """
+        if not update.effective_message:
+            self.logger.warning("Update.effective_message is empty")
+            return await self.restart(update, context)
+        if not update.effective_message.photo:
+            await update.effective_message.reply_text("Please provide photo first.")
+            return await self.restart(update, context)
+        photo = await update.effective_message.photo[-1].get_file()
+        if update.effective_user:
+            path = f"data/{update.effective_user.id}_sticker.png"
+        else:
+            path = "data/unknown_user_id_sticker.png"
+        await photo.download_to_drive(path)
+        # temporarily echo the file
+        with open(path, "rb") as photo_file:
```

Comment:
Should we close file before await?

Reply:
No, not necessary. After await file closes correctly

Reply:
Could this file be modified by other user request?